bin/run-dev

```diff
@@ -1,17 +1,31 @@
 #!/bin/bash
 
+set -e
+set +x
+export AWS_DEFAULT_REGION=us-west-2
+export REGION=$AWS_DEFAULT_REGION
+export AWS_S3_REGION=$AWS_DEFAULT_REGION
+export AWS_S3_BUCKET_NAME=civiform-local-s3
```

Comment:
Suggestion (blocking): let's put this in application.config directly.

Reply:
I put them in the `docker-compose.yaml`. Is this ok? I wasn't sure how to put them in the `application.config` file

Reply:
The config in `application.config` is overridden in sequential order.

For example, you want

```
aws.s3.region=us-west-2
aws.s3.region=${?AWS_S3_REGION}
```

The config will pick up environment variable `AWS_S3_REGION` if it is set, or use `us-west-2` if not.